### PR TITLE
Added responsiveness

### PR DESCRIPTION
**Pull request by VLeads**
This PR adds responsiveness to rapid store

Also logo has been changed.

Please review this PR and let me know your feedback and any changes required to be done.
Thankyou !!

### CODE REVIEW

File: src/pages/homepage/Homepage.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const Homepage = () => {
   return (
     <main className="homepage-main">
       <div className="hero">
-        <img src={heroPoster} alt="poster" loading="lazy" />
+        {/* <img src={heroPoster} alt="poster" loading="lazy" /> */}
```
Comment (foolhardy21) [inline src/pages/homepage/Homepage.jsx]: commented code.

**Review by foolhardy21 — APPROVED** [review]
All the logic seems good. Nothing to point out.

File: src/components/card/cart-product-card/cartProductCard.css [hunk 2]
```diff
@@ -1,54 +1,53 @@
-.card-footer{
-    margin-top:auto;
+.card-footer {
+    margin-top: auto;
```
Comment (vrx29) [inline src/components/card/cart-product-card/cartProductCard.css:2]: margin-top: doesn't work. It takes margin:0 in auto case. Please check if its working or not if not then you can remove this peice of code

**Review by vrx29 — APPROVED** [review]
Every things looks good. You can merge it. Just check one thing Is your login/signup working as it was not working for me.

**Comment by VLeads** [comment]
> Every things looks good. You can merge it. Just check one thing Is your login/signup working as it was not working for me.

Yes, it was not working as I missed to add env to netlify. Now It's fine. You can check on:  https://rapidstore-ecom.netlify.app/

File: src/components/card/cart-product-card/cartProductCard.css (hunk 2 above)
Comment (VLeads) [reply src/components/card/cart-product-card/cartProductCard.css:2]: Yes margin-top auto works. You can check it on my hosted site : https://rapidstore-ecom.netlify.app/

**Pull request merged**